command/commands/team.py

```diff
@@ -476,3 +486,59 @@ def delete_helper(self, team_name, user_id) -> ResponseTuple:
             logging.error("team delete unsuccessful")
             return f"Team delete was unsuccessful with " \
                    f"the following error: {e.data}", 200
+
+    def refresh_helper(self, user_id) -> ResponseTuple:
+        """
+        Ensure that the local team database is the same as GitHub's.
+
+        In the event that our local team database is outdated compared to
+        the teams on GitHub, this command can be called to fix things.
+        :return: return error message if user has insufficient permission level
+                 otherwise returns success messages with # of teams changed
+        """
+        num_changed = 0
+        num_added = 0
+        num_deleted = 0
+        modified = []
+        try:
+            command_user = self.facade.retrieve(User, user_id)
+            if not check_permissions(command_user, None):
+                return self.permission_error, 200
+            local_teams = self.facade.query(Team)
+            remote_teams = self.gh.org_get_teams()
+            local_ids = dict((team.github_team_id, team)
+                             for team in local_teams)
+            remote_ids = dict((team.github_team_id, team)
```

Comment:
`org_get_teams()` returns a list of `Team` objects that are from PyGithub, so they aren't using our team implementation. You should call `team.id` instead of `team.github_team_id`

Reply:
https://pygithub.readthedocs.io/en/latest/github_objects/Team.html?highlight=team

Reply:
yeet